### PR TITLE
Add new projects in FAQs

### PR DESCRIPTION
**Pull request by PremanandChowdhury**
@swarajpure Hey Check the changes please.
Thank you.

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -59,15 +59,17 @@ <h3 id="aboutRDS" class="question">How do I contribute to RDS? </h3>
             </div>
             <div class="faq__text">
               <p>
-                As of now we're having 6 projects. Every project and the corresponding tech stack is mentioned below :
+                As of now we're having 8 projects. Every project and the corresponding tech stack is mentioned below :
               </p>
               <ul class="faq__list">
                 <li><a href="https://github.com/Real-Dev-Squad/website-welcome" target="_blank" rel="noopener noreferrer">Welcome</a> : HTML, CSS, Javascript</li>
                 <li><a href="https://github.com/Real-Dev-Squad/website-members" target="_blank" rel="noopener noreferrer">Members</a> : React</li>
                 <li><a href="https://github.com/Real-Dev-Squad/rds-events-website" target="_blank" rel="noopener noreferrer">Events</a>: React</li>
```
Comment (swarajpure) [inline faq.html:67]: Can you also remove this line? This project is no more active

**Review by swarajpure — CHANGES_REQUESTED** [review]
Good job! 🎉 
Suggesting one small change

**Comment by swarajpure** [comment]
Also can you please change the sequence of the projects?
First should be plain HTML CSS
Then all the react projects
Then react + typescript
Then ember
Then nodejs

**Comment by PremanandChowdhury** [comment]
> Also can you please change the sequence of the projects?
> First should be plain HTML CSS
> Then all the react projects
> Then react + typescript
> Then ember
> Then nodejs

**Comment by PremanandChowdhury** [comment]
> > Also can you please change the sequence of the projects?
> > First should be plain HTML CSS
> > Then all the react projects
> > Then react + typescript
> > Then ember
> > Then nodejs

**Comment by PremanandChowdhury** [comment]
The requested changes are completed. Do check. 👍🏼

**Review by swarajpure — APPROVED** [review]
Perfect! Thank you! 🥳
Merging it!✌🏻

**Pull request merged**